followthemoney/types/__init__.py

```diff
@@ -22,27 +24,70 @@
 from followthemoney.types.number import NumberType
 from followthemoney.types.common import PropertyType
 
+
+class Registry(object):
+    """This registry keeps the processing helpers for all property types in the system. The
+    registry can be used to get a type, which can itself then clean, validate or format values
+    of that type."""
+
+    url = UrlType()
+    name = NameType()
+    email = EmailType()
+    ip = IpType()
+    address = AddressType()
+    date = DateType()
+    phone = PhoneType()
+    country = CountryType()
+    language = LanguageType()
+    mimetype = MimeType()
+    checksum = ChecksumType()
+    identifier = IdentifierType()
+    iban = IbanType()  # TODO: remove
```

Comment:
@pudo I know you wanted to remove the IBAN type before in favor of `identifier` with an IBAN format. Maybe 4.0 is the right time to do that (even though that will require adjustments in Aleph)?

Reply:
I think ingest-file should be fine, mostly, except that we might want to align the IBAN parsing. Looking at Aleph, indexing/cross-ref, stats/UI are probably the main areas that would need to be adjusted. I could imagine that e.g. having a list of top IBANs is still quite useful, so we might want to compute stats for IBANs even though they aren’t a separate type anymore.

Reply:
> FWIW there's a concept of "strong identifiers" no

That’s only in rigour for now, right?